src/generator/css/base.less

```diff
@@ -271,6 +336,12 @@ body {
         left: 0;
         margin-bottom: 10px;
 
+        div.action.button.primary {
+            @media (max-width: 350px) {
+                width: 12rem;
+            }
```

Comment:
Consider using tailwind statements here for the media query.

In general, when not using tailwind, I would recommend raising media queries to the top of the class structure so that we get duplicated lines.